src/views/CreateEventView.vue

```diff
@@ -14,10 +17,15 @@ import api from '../api/api';
 export default {
   name: 'CreateEvent',
   components: { EventForm },
+  data: () => ({
+    response500: false,
+  }),
   methods: {
     async createEvent({ event }) {
       const response = await api.createEvent(event);
-      if (response) {
+      if (!(response.status) || response.status >= 500) {
```

Comment:
Shouldn't this really just be if there's any error? For example, if there's a 400 bad request, this code would redirect the user anyways. Might be worth generalizing this to catch any errors.

Reply:
Yeah I wasn't sure about this. I figured we were catching possible 400 responses with validation but you're right we don't wan to redirect anyway in the event something weird happens. Will change.